tests/test_group.py

```diff
@@ -0,0 +1,117 @@
+
+        # Update the group description fields
+        group_description.set_description_text(new_group_description_text)
+        group_description.set_irc_channel(new_group_irc_channel)
+        group_description.click_update()
+
+        search_listings = home_page.header.search_for(group_name)
+        assert search_listings.is_element_present(By.LINK_TEXT, group_name)
+        group_info = search_listings.open_group(group_name)
+
+        # Check that everything was updated
+        assert new_group_description_text == group_info.description_text
+        assert new_group_irc_channel == group_info.irc_channel
+
+        # Delete group
+        group_info.delete_group()
+
+    @pytest.mark.credentials
+    def test_group_deletion_confirmation(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
+
+        delete_form = group.description.delete_group
+
+        assert not delete_form.is_delete_button_enabled
+
+        delete_form.check_acknowledgement()
+
+        assert delete_form.is_delete_button_enabled
+
+        groups_page = delete_form.click_delete_group()
+        assert groups_page.is_group_deletion_alert_present
+
+    @pytest.mark.credentials
+    def test_group_type_change(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
+
+        group_type = group.access.group_type
+
+        assert not group_type.is_member_criteria_visible
+
+        group_type.set_reviewed_group_type()
+
+        assert group_type.is_member_criteria_visible
+
+    @pytest.mark.credentials
+    def test_group_invitations(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
```

Comment:
This block is used a lot, I wonder if we should add a `create_group` method to `SettingsPage` to reduce some duplication.